package.json

```diff
@@ -33,29 +33,33 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "@babel/generator": "7.17.7",
-    "@babel/parser": "^7.20.5",
-    "@babel/traverse": "7.23.2",
-    "@babel/types": "7.17.0",
+    "@babel/generator": "7.26.2",
+    "@babel/parser": "7.26.2",
+    "@babel/traverse": "7.25.9",
+    "@babel/types": "7.26.0",
     "javascript-natural-sort": "0.7.1",
     "lodash": "^4.17.21"
   },
   "devDependencies": {
-    "@babel/core": "^7.20.7",
-    "@types/chai": "4.2.15",
-    "@types/jest": "26.0.20",
-    "@types/lodash": "4.14.168",
-    "@types/node": "20.8.6",
-    "@vue/compiler-sfc": "^3.2.41",
-    "jest": "26.6.3",
-    "prettier": "2.8",
-    "ts-jest": "26.5.3",
-    "typescript": "4.9.4"
+    "@babel/core": "7.26.0",
+    "@types/chai": "5.0.1",
+    "@types/jest": "29.5.14",
+    "@types/lodash": "4.17.13",
+    "@types/node": "22.9.0",
+    "@vue/compiler-sfc": "^3.5.12",
+    "jest": "29.7.0",
+    "prettier": "3.3.3",
+    "ts-jest": "29.2.5",
+    "typescript": "5.6.3"
   },
   "peerDependencies": {
     "@vue/compiler-sfc": "3.x",
     "prettier": "2.x - 3.x"
   },
+  "engines": {
+    "node": ">18.12"
+  },
+  "packageManager": "yarn@4.5.1",
```

Comment:
The `packageManager` field is in experimental. I would recommend to not to add. https://nodejs.org/docs/latest-v20.x/api/all.html#all_packages_packagemanager

Reply:
I've deleted "packageManager" from package.json and removed `corepack` from CI since packageManager was mandatory thing for using it. However, I would like to notice the fact that although all these features are experimental, they have been around for 4 years.

I should ask you about `--experimental-vm-modules` which I added for launching tests with the newest prettier version and it was necessary since prettier@3 utilizes dynamic import. I tried quickly transpile code to omit `--experimental-vm-modules`, but failed. Should I to figure out how to do so without `--experimental-vm-modules`?